### ISSUE TITLE
"Everything" view doesn't work when there are three or more instances

### ISSUE THREAD

**Issue by kuba-orlik** [krawieck/lemmur#183]
### Device info

- OS: Android 8.1.0, this issue also happens on Linux
- Device: LG Nexus 5X

### Describe the bug
When being logged in to two accounts (three counting the unremovable lemmy.ml) the main home view doesn't work. I can see each individual instance, but not the Everything view

### Steps to reproduce

1. Log in to two accounts
2. Click on the home icon in bottom left
3. Choose any option under "Everything" in the dropdown on the top
4. See an endless spinner

### Expected behavior
I expect to see entries from every instance

### Screenshots/Screencasts
https://video.kuba-orlik.name/videos/watch/1288df41-180a-4f93-a15c-f8eca519edef

**Comment by shilangyu** [comment]
Are you by any chance subscribed to 0 communities on one of those instances? I found a bug where we incorrectly make an assumption that each instance has at least one community being followed. If that is not the case for you I will keep looking for other sources of the infinite spinner.

> unremovable lemmy.ml

What do you mean? lemmy.ml can be removed, it is only included by default?

**Comment by kuba-orlik** [comment]
> Are you by any chance subscribed to 0 communities on one of those instances? 

That is correct. That instance is lemmy.ml, which I didn't log into in the app.



> > unremovable lemmy.ml

> What do you mean? lemmy.ml can be removed, it is only included by default

If so, then I have no idea how to do it. All I see is to add an account to lemmy.ml, no option to remove lemmy.ml from the list of instances:

![image](https://user-images.githubusercontent.com/2697916/111805331-32fe4880-88d1-11eb-8598-446c39f74f13.png)

**Comment by shilangyu** [comment]
We should make it more obvious, sorry for that: you can swipe the instance to the left to reveal a remove button

**Comment by kuba-orlik** [comment]
I would never have guessed that 😅
 
 I think the delete button could just be there without having to swipe in order to reveal it

**Comment by kuba-orlik** [comment]
Now I'm logged into three instances and I *am* subscribing at least one community on each of them. The issue with Everything view not loading still persists. I guess it wasn't caused by having 0 subscriptions in one of the instances

**Comment by shilangyu** [comment]
Please let me know if this issue still persist in `v0.4.1`

**Comment by krawieck** [comment]
@kuba-orlik does the issue persist?

**Comment by kuba-orlik** [comment]
I've installed 0.4.2 and I can't add any of my instances, it says "instance not found", so I can't test it

**Comment by shilangyu** [comment]
That is because szmer.info, board.sealcode.org, and poznan.group all run outdated versions of Lemmy (the API changed)
